docs/developers/guides/automatic_coin_conversion.md

```diff
@@ -0,0 +1,81 @@
+	// the port on which the packet will be sent
+	SourcePort string `protobuf:"bytes,1,opt,name=source_port,json=sourcePort,proto3" json:"source_port,omitempty" yaml:"source_port"`
+	// the channel by which the packet will be sent
+	SourceChannel string `protobuf:"bytes,2,opt,name=source_channel,json=sourceChannel,proto3" json:"source_channel,omitempty" yaml:"source_channel"`
+	// the tokens to be transferred
+	Token types.Coin `protobuf:"bytes,3,opt,name=token,proto3" json:"token"`
+	// the sender address
+	Sender string `protobuf:"bytes,4,opt,name=sender,proto3" json:"sender,omitempty"`
+	// the recipient address on the destination chain
+	Receiver string `protobuf:"bytes,5,opt,name=receiver,proto3" json:"receiver,omitempty"`
+	// Timeout height relative to the current block height.
+	// The timeout is disabled when set to 0.
+	TimeoutHeight types1.Height `protobuf:"bytes,6,opt,name=timeout_height,json=timeoutHeight,proto3" json:"timeout_height" yaml:"timeout_height"`
+	// Timeout timestamp in absolute nanoseconds since unix epoch.
+	// The timeout is disabled when set to 0.
+	TimeoutTimestamp uint64 `protobuf:"varint,7,opt,name=timeout_timestamp,json=timeoutTimestamp,proto3" json:"timeout_timestamp,omitempty" yaml:"timeout_timestamp"`
+	// optional memo
+	Memo string `protobuf:"bytes,8,opt,name=memo,proto3" json:"memo,omitempty"`
+}
+
+type Coin struct {
+	Denom  string `protobuf:"bytes,1,opt,name=denom,proto3" json:"denom,omitempty"`
+	Amount Int    `protobuf:"bytes,2,opt,name=amount,proto3,customtype=Int" json:"amount"`
+}
+```
+
+## Inbound transactions
+
+Your users may want to move IBC Coins from other Cosmos chains onto Evmos. To use these IBC coins on dApps deployed on Evmos, they need an ERC-20 representation of these. The automated coin conversion feature automatically converts the incoming IBC coins into their ERC-20 representation. In this way, you don't need to manually convert the incoming IBC coins into ERC-20 tokens. As a result, your users can use the IBC coins as ERC-20 tokens as soon as they arrive to their wallets.
+
+Should consider that only the registered token pairs are converted. If the token pair is not registered, users will receive the corresponding IBC coin on their wallet without any further change.
+
+:::tip
+**Note**: If your users have some IBC coins on Evmos already, and the token pair is registered, when they receive an IBC transfer of this denomination, their **whole balance** will be converted (the current balance plus the transfer amount).
+:::
+
+## FAQ
+
+### How do I send an ERC-20 via IBC?
+
+With the new automated coin conversion feature, you can send ERC-20 via IBC right away. The conversion step is done automatically under the hood. To do this operation you only need to specify the corresponding denomination on the `MsgTransfer` struct. For example, if we want to send an ERC-20 token called `TestCoin` via IBC, use `Token.Denom = "TestCoin"`.
+
+### Can I send WEVMOS to other chains?
```

Comment:
Are these WEVMOS or wEVMOS?

Reply:
The [assets page](https://app.evmos.org/assets) shows it as `WEVMOS`